eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp.hs

```diff
@@ -30,6 +31,7 @@ spec ::
   Spec
 spec = do
   describe "ConwayImpSpec" $ withImpState @era $ do
+    BabbageTxSpec.spec @era
```

Comment:
Good idea, but not here. For consistency sake, let's add the whole of Babbage imp spec here: https://github.com/IntersectMBO/cardano-ledger/blob/829386ef553a8d86c29fbd1ede9ad3b3b78b3659/eras/conway/impl/test/Main.hs#L30-L32
```suggestion
```